src/debug_menu.cpp

```diff
@@ -1315,9 +1357,38 @@ void debug()
                 }
                 break;
             case DEBUG_TEST_WEATHER:
+            {
                 weather_generator weathergen;
                 weathergen.test_weather();
+            }
                 break;
+
+
+            case DEBUG_SAVE_SCREENSHOT: {
```

Comment:
Inconsistent brace placement between `DEBUG_TEST_WEATHER` and here.  (I'm not sure which is the norm in the rest of the codebase).

Reply:
I fixed it by using the scheme as the other `case` in the file where the opening curly bracket `{` is on the same line as the `case`.